kubernetes/ansible/roles/deploy-player/tasks/main.yml

```diff
@@ -33,7 +33,7 @@
   shell: "kubectl create configmap player-config --from-env-file={{role_path}}/templates/{{ release_name }}.env -n {{namespace}} --dry-run -o=yaml | kubectl apply -f -"
 
 - name: helm install and upgrade
-  shell: helm upgrade --install --cleanup-on-fail {{ release_name }} {{ chart_path }} -n {{namespace}}
+  shell: helm upgrade --install --atomic --cleanup-on-fail {{ release_name }} {{ chart_path }} -n {{namespace}}
```

Comment:
@SMY-ALTAMASH  --atomic if set, upgrade process rolls back changes made in case of failed upgrade. So I don't think we need --cleanup-on-fail